flink-connector-oceanbase/src/main/java/com/oceanbase/connector/flink/OceanBaseConnectorOptions.java

```diff
@@ -118,6 +118,13 @@ public class OceanBaseConnectorOptions extends ConnectorOptions {
                     .defaultValue(Duration.ofSeconds(30))
                     .withDescription("Client heartbeat timeout in direct load task.");
 
+    public static final ConfigOption<Boolean> ORACLE_TENANT_CASE_INSENSITIVE =
+            ConfigOptions.key("oracle.tenant.case.insensitive")
```

Comment:
It's better to use a prefix with `table`, like `table.case-insensitive`, as there may be other table options in the future.